next-app/src/routes/projects/[project_code]/index.ts

```diff
@@ -0,0 +1,31 @@
+import { get_activities } from './activities'
+import { get as get_project_info } from './meta'
+
+export async function GET({ params: { project_code }, request: { headers }}) {
```

Comment:
I was wondering what the significance of capital GET and PUT is, compared to their lowercase equivalents?  Is this following a convention?

Reply:
that is a recent breaking change to SvelteKit.  They've decided to following the HTTP convention on these.  You'll notice my data wrappers were already doing this just because I personally like the matchup with HTTP and I wanted to be able to use `DELETE` in its full form rather than `del`.  Here's a broader conversation on it for your reference:  https://github.com/sveltejs/kit/discussions/5359

Reply:
ok thanks for the info!